tests/mocks/model_mock.py

```diff
@@ -57,3 +57,4 @@ def predict(self, sample: InputSample) -> List[str]:
             return sample.tags
         else:
             return ["O" for i in range(len(sample.tags))]
+            
```

Comment:
Once the new structure is in place, we would need to support it with unit tests (especially for the evaluation logic)